pkgs/applications/networking/mailreaders/himalaya/default.nix

```
@@ -46,7 +48,7 @@ rustPlatform.buildRustPackage rec {
     description = "CLI email client written in Rust";
     homepage = "https://github.com/soywod/himalaya";
     changelog = "https://github.com/soywod/himalaya/blob/v${version}/CHANGELOG.md";
-    license = licenses.bsd3;
+    license = licenses.bsd4;
```

```suggestion
    license = licenses.bsdOriginal;
```

We use `licenses.bsdOriginal` for BSD-4-Clause and `licenses.bsdOriginalUC` for BSD-4-Clause-UC.